src/Components/Products/ProductDetails.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+import { FcPrevious } from 'react-icons/fc';
+
+const ProductDetails = () => {
+  const location = useLocation();
+  const product = location.state;
+
+  return (
+    <div className="mx-5">
+      <div>
+        <NavLink to="/">
+          {' '}
+          <FcPrevious />
+          {' '}
+        </NavLink>
+      </div>
+      <div className="lg:flex p-5">
+        <div>
+          <img src={product.image} alt={product.title} />
+        </div>
+        <div>
+          <h2 className="font-extrabold text-xl my-4">{product.title}</h2>
+          <p className="">{product.category}</p>
+          <p className="font-extrabold my-3 text-xl">
+            $
+            {' '}
+            {product.price}
+            .00
+          </p>
+          <p>{product.description}</p>
+          <button type="button" className="rounded-none bg-green-700 px-2 py-1 my-3 uppercase font-bold text-white">Add To Cart</button>
+        </div>
+      </div>
+
+    </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
  -  Typography: font face, size, and weight
  -  Layout: composition and space between elements.
**Here is how look the detail pages :point_down:** 

| Template design | Your Design |
| --------------------- | ------------------- |
| ![Screenshot from 2023-01-12 23-41-24](https://user-images.githubusercontent.com/83042742/212187939-802e7094-cf91-4580-9df1-b700b9494b7d.png) | ![Screenshot from 2023-01-12 23-45-50](https://user-images.githubusercontent.com/83042742/212187966-3aa1c727-ac2c-41c5-8b53-0794c613d6b7.png) |